src/lua/log.lua

```diff
@@ -395,6 +396,63 @@ local function log_pid()
     return tonumber(ffi.C.log_pid)
 end
 
+local Ratelimit = {
+    interval = 60,
+    burst = 10,
+    emitted = 0,
+    suppressed = 0,
+    start = 0,
+}
+
+function Ratelimit:new(object)
+    object = object or {}
+    setmetatable(object, self)
+    self.__index = self
+    return object
+end
+
+function Ratelimit:check()
+    local clock = require('clock')
```

Comment:
Nit: it's better to require it once.

Reply:
Fixed: moved the require to the beginning of the module file.

Reply:
Actually, I can't move it out because of the order in which modules are loaded:
https://github.com/tarantool/tarantool/blob/12cf9980eaa9542bf222d4ae7dd5aad0ccf218cc/src/lua/init.c#L246-L267

`clock` module is loaded after `log` module.